docs/sphinx/cookbook/shared_memory_allocators.rst

```diff
@@ -74,14 +74,22 @@ uses the MPI3 API for its Shared Memory mechanisms that allow processes to commu
 To use Umpire's MPI3 Shared Memory allocators, the ``UMPIRE_ENABLE_MPI3_SHARED_MEMORY`` flag 
 should be set to ``On``. Note that if you are using MPI3 Shared Memory, then MPI must be enabled.
 
-See the bottom of this page for a full example of how to use Shared Memory Allocators with Umpire.
+See the bottom of this page for a full example of how to use MPI3 Shared Memory Allocators with Umpire.
 
-Full IPC Shared Memory Recipe
------------------------------
+Ful Shared Memory Examples
+--------------------------
 
-The following example shows how to create, use, and destruct the IPC Shared Memory Allocator whether using MPI or not. For an example of using
-MPI3 Shared Memory Allocators, users could simply update the included header file. No other code changes are necessary assuming the input
-parameter specifies MPI for the following example code.
+This section shows two full code examples, one for IPC Shared Memory and one for MPI3.
+
+The following example shows how to create, use, and destruct the IPC Shared Memory Allocator whether using MPI or not. Note that this
```

Comment:
I am a bit confused by the statement "whether using MPI or not". That could probably be left off, or made into a separate note saying that IPC shared memory can be used even when MPI and/or MPI3 shared memory are enabled.

Reply:
Made a note for this.